packages/suite-desktop/e2e/tests/coinjoin.test.ts

```diff
@@ -61,15 +73,19 @@ const addCoinjoinAccount = async (window: Page) => {
     await window.click('[data-test="@add-account-type/select/option/Bitcoin Regtest (CoinJoin)"]');
     await window.click('[data-test="@add-account"]');
 
+    await window.screenshot({ path: './test-results/1.png', fullPage: true, type: 'png' });
     await window.click('[data-test="@request-enable-tor-modal/skip-button"]');
-
+    await window.screenshot({ path: './test-results/2.png', fullPage: true, type: 'png' });
     await window.waitForSelector('[data-test="@prompts/confirm-on-device"]', {
         timeout: 60 * 1000,
     });
+    // todo: this is weird it looks like device sometimes is not ready to accept pressYes, although modal
+    // is already visible
+    await window.waitForTimeout(5000);
```

Comment:
501 is not enough? maybe bridge didn't finished enumeration

Reply:
not sure, could be. these todos are future issues to be solved. timeout shouldnt be needed here at all.